### PR TITLE
Added link to original toot on date

### PR DESCRIPTION
**Pull request by nicosomb**

### CODE REVIEW

File: src/View.elm [hunk 1]
```diff
@@ -239,7 +240,7 @@ accountTimelineView account statuses label iconName =
 statusActionsView : Mastodon.Status -> Html Msg
 statusActionsView status =
     let
-        target =
+        tootTarget =
```
Comment (n1k0) [inline src/View.elm]: I think the general consensus in the code has been to use `status` for statuses, not `toot`. While I think `target` was fine, if you still want to precise its nature let's use `targetStatus` instead, for the sake of consistency.

File: src/View.elm [hunk 2]
```diff
@@ -248,18 +249,18 @@ statusActionsView status =
...
             case status.favourited of
                 Just True ->
-                    ( baseBtnClasses ++ " favourited", RemoveFavorite target.id )
+                    ( baseBtnClasses ++ " favourited", RemoveFavorite tootTarget.id )
 
                 _ ->
-                    ( baseBtnClasses, AddFavorite target.id )
+                    ( baseBtnClasses, AddFavorite tootTarget.id )
 
         tootDate =
```
Comment (n1k0) [inline src/View.elm]: So please also update `tootDate` to be `statusDate` 🙂

File: src/View.elm (hunk 1 above)
Comment (nicosomb) [inline src/View.elm]: Fixed. Squash?

Comment (n1k0) [inline src/View.elm]: I prefer tennis.

(Landed, thanks!)

**Pull request merged**